docs/pages/blog/mui-x-v8-alpha-zero.md

```diff
@@ -0,0 +1,96 @@
+---
+title: The road to MUI X v8
+description: Explore the planned features and our roadmap to the next major version.
+date: 2024-11-20T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'Product']
+manualCard: true
+---
+
+We're kicking off the development of [MUI X v8](https://github.com/mui/mui-x/releases/tag/v8.0.0-alpha.0).
+Following our yearly release cycle, the target for the first stable release is March 2025.
+This major update includes new versions of the Data Grid, Charts, Tree View, and Date & Time Pickers.
+We’re excited to share our roadmap with you and invite you to join us on this journey!
+
+:::warning
+Only MUI X is getting a new version—Material UI will remain in v6 for now.
+You can rest assured that all MUI X v8 components will be fully compatible with Material UI v5 and v6.
```

Comment:
```suggestion
Since only MUI X is getting a new version, you can rest assured that all MUI X v8 components will be fully compatible with Material UI v5 and v6.
```